packages/protocol/src/action-protocol/smart-connector.ts

```diff
@@ -0,0 +1,65 @@
+/********************************************************************************
+ * Copyright (c) 2023 Business Informatics Group (TU Wien) and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import { hasStringProp } from '../utils/type-util';
+import { Action } from './base-protocol';
+
+/**
+ * Action that opens the smart connector at the position of the element
+ */
+export interface OpenSmartConnectorAction extends Action {
+    kind: typeof OpenSmartConnectorAction.KIND;
+
+    /**
+     * The identifier of the element where the smart connector is to be opened.
+     */
+    selectedElementID: string;
```

Comment:
```suggestion
    selectedElementId: string;
```

The code base uses `Id` usually.